assets/aws/files/bin/teleport-all-pre-start

```diff
@@ -10,11 +11,29 @@ if [ -f /etc/teleport.d/conf ]; then
     source /etc/teleport.d/conf
 fi
 
-# check for Let's Encrypt
+# Check for Let's Encrypt
 if [[ "${USE_LETSENCRYPT}" != "true" ]]; then
     echo "Not using Let's Encrypt, exiting with success"
     exit 0
 fi
 
-# copy certificates into place
-/bin/aws s3 sync --exact-timestamps s3://${TELEPORT_S3_BUCKET}/live/${TELEPORT_DOMAIN_NAME} /var/lib/teleport
+# Copy certificates into place
+# Exclude debug.sock and any other .sock files from the sync
+# This is necessary because the sync operation will fail if the debug.sock file is present
+/bin/aws s3 sync \
+    --exact-timestamps \
+    --exclude="debug.sock" \
+    --exclude="*.sock" \
```

Comment:
```suggestion
```

I'd remove this. The only file we know that causes a problem is debug.sock - let's not over-generalize the solution.